src/content/en/updates/2017/01/css-grid.md

```diff
@@ -0,0 +1,225 @@
+Flexbox made a lot of complex layout tasks easier, like equi-distant spacing
+between elements, top-to-bottom layouts or the holy grail of CSS wizardry:
+vertical centering.
+
+<img width="50%" src="/web/updates/images/2017/01/css-grid/alignproblem.png" alt="There's no way to align elements across multiple flexbox containers.">
+
+But alas, screens commonly have a second dimension we need to worry about.
+Short of taking care of sizing the elements yourself, sadly, you can't have
+*both* a vertical and horizontal rhythm by just using flexbox alone. This is
+where CSS Grid comes to the rescue.
+
+CSS Grid has been in development, behind a flag in most browsers for over
+*5 years* now and extra time has been spent on interoperability to avoid the
+buggy launch like Flexbox had. So if you use Grid to implement your layout in
+Chrome, chances are you'll get the same result in Firefox and Safari. At the
+time of writing, Microsoft's Edge implementation of Grid is out of date (the
+same as was already present in IE11!) and the update is
+"[under consideration](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/gridupdate/)".
+
+Despite the similarities in concept and syntax, don't think of Flexbox and Grid
+as competing layout techniques. Grid arranges in two dimensions, while Flexbox
+lays out in one and the both have a lot of synergy when used together.
+
+## Defining a grid
+
+To get familiar with the individual properties of Grid I heartily recommend
+\Rachel Andrew’s [Grid By Example](http://gridbyexample.com) or CSS Tricks'
+[Cheat Sheet](https://css-tricks.com/snippets/css/complete-guide-grid/). If you
+are familiar with Flexbox, a lot of the properties and their meaning should be
+familiar.
+
+Let's take a look at a standard 12-column grid layout. The classic 12-column
+layout is very popular as the number 12 is divisible by 2, 3, 4 and 6 and is
+therefore useful for a lot different designs. Let's implement this layout:
+
+<img width="50%" src="/web/updates/images/2017/01/css-grid/examplelayout.png" alt="There's no way to align elements across multiple flexbox containers.">
+
+Let's start with our markup code:
+
+    <!DOCTYPE html>
+    <body>
+      <header></header>
+      <nav></nav>
+      <main></main>
+      <footer></footer>
+    </body>
+
+In our stylesheet we start by expanding our `body` so it covers the entire
+viewport and turning it into a *grid container*:
+
+    html, body {
+      width: 100vw;
+      min-height: 100vh;
+      margin: 0;
+      padding: 0;
+    }
+    body {
+      display: grid;
+    }
+
+Now we are using CSS Grid. Hooray!
+
+The next step is to implement the rows and columns of our grid. We could
+implement all 12 columns in our mockup, but as we are not using every column,
+doing this would make our CSS unnecessarily messy. For the sake of simplicity,
+we'll implement the layout this way:
+
+<img width="50%" src="/web/updates/images/2017/01/css-grid/simplifiedexamplelayout.png">
```

Comment:
Don't include widths here, if you want this floated left or right, use `class="attempt-left|right"`